README.md

```diff
@@ -1,10 +1,10 @@
 # Timed Content For Beaver Builder #
 
-**Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce)
+**Contributors:** [pratikchaskar](https://profiles.wordpress.org/pratikchaskar/)
```

Comment:
Did we made these changes in last update? If we manually added bsf name here in the last update then only make these changes

Reply:
No, we didn't do any changes in last update. 
It was redirecting to the below earlier: 
`**Contributors:** [brainstormforce](https://profiles.wordpress.org/brainstormforce)`